---
title: "*Orchid fleck dichorhavirus*: a new *Brevipalpus* transmitted virus fresh from Florida"
subtitle: "A. **Fife**^1^, D. **Carrillo**^2^, G. **Knox**^3^, F. **Iriarte**^4^, K. **Dey**^5^, A. **Roy**^6^, R. **Ochoa**^7^, G. **Bauchan**^8^, M. **Paret**^4,9^, X. **Martini**^1^*"
author: "^1^ University of Florida, Department of Entomology and Nematology, North Florida Research and Education Center, Quincy FL 32351 ^2^ University of Florida, Department of Entomology and Nematology, Tropical Research and Education Center, Homestead FL 33031 ^3^ University of Florida, Department of Environmental Horticulture, North Florida Research and Education Center, Quincy FL 32351 ^4^ University of Florida, Department of Plant Pathology, North Florida Research and Education Center, Quincy FL 32351 ^5^ The Florida Department of Agriculture and Consumer Services, Division of Plant Industry, Section of Plant Pathology, Doyle Conner Building, 1911 SW 34th street, Gainesville, FL 32608 ^6^	United States Department of Agriculture – Agriculture Research Service, Molecular Plant Pathology Laboratory, 10300 Baltimore Ave, Bldg. 4 BARC-West, Beltsville, MD 20705 ^7^ United States Department of Agriculture - Agriculture Research Service, Systematic Entomology Laboratory 10300 Baltimore Ave, Bldg. 5 BARC-West, Beltsville, MD 20705 ^8^	United States Department of Agriculture - Animal and Plant Health Inspection Service, Electron and Confocal Microscopy Unit, Bldg. 12  BARC-West, 10300 Baltimore Ave, Beltsville, MD 20705 ^9^ Plant Pathology Department, University of Florida, Gainesville, FL 32611"
date: March 29, 2021
output: 
  powerpoint_presentation
bibliography: 'thesis.bib'
csl: 'j_med_entomol.csl'
always_allow_html: true
---
```{r setup, include=FALSE}
#setup options
knitr::opts_chunk$set(fig.width=12, fig.height=8, echo=FALSE)

#a list of packages used for scripts
pkgs <-
  c(
    'captioner',
    'pryr',
    'tiff',
    'sf',
    'rgeos',
    'ggspatial',
    'rnaturalearth',
    'rnaturalearthdata',
    'tidyverse',
    'ggmap',
    'ggplot2',
    'ggthemes',
    'extrafont',
    'showtext',
    'leaflet',
    'dplyr',
    'readxl',
    'maps',
    'mapview',
    'htmlwidgets',
    'viridis',
    'RColorBrewer'
  )

#loading required packages
lapply(pkgs, library, character.only = TRUE)
rm(pkgs)

#setting up captions for use in the document
fig_nums <- captioner::captioner(prefix = 'Fig.')
tab_nums <- captioner::captioner(prefix = 'Table')

#allows me to use a custom function instead of the citation function of captioner, which is wordy to use
citef <- pryr::partial(fig_nums, display = "cite")
citet <- pryr::partial(tab_nums, display = "cite")
```

# Chlorotic ringspot symptoms on Giant Lilyturf (*Liriope* spp., cv. ‘Gigantea’)
```{r, warning=FALSE}
knitr::include_graphics('images/symp-ofv-lir-1.jpg')
```

# Chlorotic leaves on Cast Iron Plant (*Aspidistra elatior*)
```{r, warning=FALSE}
grid::grid.raster(tiff::readTIFF("images/symp-ofv-asp-1.tif"))
```

# Chlorotic spots on *A. elatior* - Early infection?
```{r, warning=FALSE}
grid::grid.raster(tiff::readTIFF("images/symp-ofv-asp-2.tif"))
```

# What was causing these symptoms?
:::::::::::::: {.columns}
::: {.column}
 - Tested negative for begomovirus, potyvirus, tospovirus as well as for *Impatiens necrotic spot virus*, *Tobacco mosaic virus* and *Tomato spotted wilt virus*
 - Sent to the Florida Department of Agriculture and Consumer Services (FDACS) $\rightarrow$ sent to USDA
:::
::: {.column}
![Symptoms on varigated Ophiopogonae](images/symp-ofv-lir-2.jpg)
:::
::::::::::::::

# Confirmation: *Orchid fleck dichorhavirus* (OFV)
 - Identified using OFV specific conventional reverse transcription polymerase chain assay (RT-PCR) assay + Sanger sequencing + quantitative RT-PCR (RT-qPCR)
 - Presence of Orchid fleck dichorhavirus confirmed using OFV generic R2-Dicho-GF and R2-Dicho-GR primers [@Roy2020] 
 - Sanger sequencing of RT-PCR amplicons shared 98\% nucleotide identity with orchid strains of OFV: OFV-Orc1 and OFV-Orc2 (GenBank Accession numbers: AB244418 and LC222630) [@Kondo2006; @Kondo2017].

# What is OFV?
:::::::::::::: {.columns}
::: {.column}
 - Type member for *Dichorhavirus*, family Rhabdoviridae
 - Other dichorhaviruses: *Citrus chlorotic spot virus*, *Citrus leprosis virus* N, *Clerodendrum chlorotic spot virus* and *Coffee ringspot virus*
 - OFV infects 50+ plant species, including Orchidaceae, Asparagaceae (Nolinoidaea), and Rutaceae (*Citrus*)
 - Flat mites from the genus *Brevipalpus* Donnadieu (Trombidiformes: Tenuipalpidae) only known vectors of dichorhaviruses
:::
::: {.column}
![Symptoms of OFV on *Oncidium*](images/oncidium_ofv.tif)
:::
::::::::::::::

# Worldwide Distribution of OFV
 - First described infecting *Cymbidium* orchids in Japan [@Doi1977]
 - OFV and OFV-like rhabdoviruses have been reported worldwide
 - Asia, Africa, North America, South America, Europe and Oceania
 - Prevalence of OFV and its mite vector is thought to be associated with the movement of infected orchids [@Dietzgen2018a]

# World Map of OFV Distribution
```{r, warning=FALSE, echo=FALSE, comment=FALSE}
#https://slcladal.github.io/maps.html
library(maptools)
data(wrld_simpl)
myCountries = wrld_simpl@data$NAME %in% c("China", "Korea, Republic of", "Korea, Democratic People's Republic of", "South Africa", "United States", "Mexico", "Brazil", "Colombia", "Costa Rica", "Paraguay", "Denmark", "France", "Germany", "Australia", "Fiji", "Vanuatu", "Japan")
plot(wrld_simpl, col = c(gray(.90), "orange")[myCountries+1])
```

# Map of OFV distribution in Tallahassee 
```{r}
#####LOADING REQUIRED FILES####
df <- read_xlsx('data/ofv_clean_datasheet.xlsx')

#making sure R knows what variables are factors
df$symptoms <- as_factor(df$symptoms)
df$plant_cv <- as_factor(df$plant_cv)
df$family <- as_factor(df$family)
df$order <- as_factor(df$order)

####REMOVING NAS####
#removing columns of NA
df <- drop_na(df)

#makes the dataframe into a spatial dataframe
df <-
  st_as_sf(df,
           coords = c("lon", "lat"),
           crs = 4326)

####GRAPHING SAMPLING LOCATIONS####
#creating a subset of the data containing only asparagales
asp <- df %>%
  filter(order == 'asparagales')

#making color pallet for the map
#the order goes (ringspot, stippling, none, chlorosis)
pal <- c('#092C48', '#A3B7F9', '#EDE6DE', '#C59434')

#mapping locations
mapview(
  asp,
  zcol = 'symptoms',
  col.regions = pal,
  legend = TRUE,
  alpha.regions = 1,
  layer.name = 'Ringspot Symptoms on Asparagales',
  map.types = 'Stamen.TonerLite'
)
```

# Mites Recovered During Our Survey
:::::::::::::: {.columns}
::: {.column}
- Unidentified eriophyid mite
- Three *Brevipalpus* mite species were recovered from OFV-infected plants: 
  - *Brevipalpus californicus* sensu lato
  - *B. obovatus* Donnadieu 
  - *B. confusus* Banks
- One of these species is presumably responsible for OFV transmission
- **Cryo-SEM to help determine *Brevipalpus* species identity**
:::
::: {.column}
![Various *Brevipalpus* spp. live in Florida](images/other_mites.tif)
:::
::::::::::::::

# The Main Suspect: *B. californicus* sensu lato
![*Brevipalpus californicus* s.l](images/b_cali_detail.tif)
- *Brevipalpus californicus* (Banks) group are the only known to transmit OFV in a persistent propagative manner
  - *Brevipalpus* mite species complex contains cryptic species $\rightarrow$ develop molecular methods?

# OFV is an issue for Florida agriculture
:::::::::::::: {.columns}
::: {.column}
 - The dichorhavirus that infects citrus in Hawaii, Mexico, Colombia, and South Africa are identical to the OFV in gene order, content, and the genome sequence.
 - ICTV: Officially, OFV consists of two orchid strains (OFV-Orc1 and OFV-Orc2) and two citrus strains (OFV-Cit1 and OFV-Cit2)
 - Orchid strains of OFV infects citrus [@Roy2020], but none of the citrus strains have been reported from any orchid species
 - OFV in Florida represents a concern for horticulturists who grow orchids, *Liriope*, *Ophiopogon*, or other susceptible Asparagaceae species which are commonly used in landscaping
:::
::: {.column}
```{r}
grid::grid.raster(tiff::readTIFF("images/NPA-20-02-Orchid-fleck-virus_Page_1.tiff"))
```
:::
::::::::::::::

# Acknowledgements
We would like to give a special thanks to the Tallahassee Museum for their patience, cooperation, and support with collecting plant samples. We also want to thank Drs. Sam Bolton, FDACS and Aline Tassi, Univ. of Sao Paulo, Brazil for checking the mites we have sent for species validation. We are especially indebted to the late Dr. Gary Bauchan for his contributions to this study and the field of acarology, he will be greatly missed. This research was partly funded by the USDA National Institute of Food and Agriculture, Hatch project FLA-NFC-005607. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the USDA; USDA is an equal opportunity provider and employer.


# References
<div id="refs"></div>